tark/tark/templates/documentation.html

```diff
@@ -58,148 +135,3640 @@ <h4 id="tark_we">What services are provided by Web Tark?</h4>
+					
+					<!-- Remove GET and provide a link to REST -->
+					<!-- <h4 id="tark_crud">How to do GET operations via REST API?</h4>
+					<div class="panel panel-info ">
+					<div class="panel-heading">GET</div>
+					<div class="panel-body">
+					curl -X GET --header 'Accept: application/json' '{{hostname}}/api/transcript/'<br/>
+					curl -X GET --header 'Accept: application/json' '{{hostname}}/api/transcript/?stable_id=ENST00000639989'<br/>
+					curl -X GET --header 'Accept: application/json' '{{hostname}}/api/transcript/?stable_id=ENST00000639989&assembly_name=GRCh38'<br/>
+					curl -X GET --header 'Accept: application/json' '{{hostname}}/api/transcript/?stable_id=ENST00000639989&assembly_name=GRCh38&release_short_name=90'<br/><br/>
+					</div>
+					</div> 
+					<hr/>
+					<h4 id="ar_tech_stack">What is the technology stack of Tark?</h4>
+					<p>  
+					<a href="https://www.python.org/">Python3</a>, <a href="https://www.djangoproject.com/">Django</a>, <a href="http://www.django-rest-framework.org/">Django REST Framework</a> and <a href="https://django-rest-swagger.readthedocs.io/en/latest/">Django REST Swagger</a>. Backend support is provided by<a href="https://www.mysql.com/"> MySQL</a>.
+					</p>
+					<a href="#tark_summary">Go Top</a>
+					 </div>
+					</div>
+					</div>
+					</div>
+					</div>	
+					-->
+					<h4 id="tark_crud">How to do GET operations via REST API?</h4>
+					<p>Please visit <a href="http://tark.ensembl.org/api/">REST API Endpoints</a>. Description and examples of each request are given below.</p>
+					<!--------------------------------------------------------ASSEMBLY----------------------------------------------------------------------------------------->	
+					<div class="collapsible">- Assembly</div>
+					<div class="content">
+						<table class="table table-striped" style="font-size:14px">
+						<thead>
+						<tr><th>Resource</th><th>Description</th><th>Example Requests</th></tr>
+						</thead>
+							<tbody>	              
+							   <tr><td><a href="http://tark.ensembl.org/api/#!/assembly/assembly_list">GET /api/assembly/ </a> </td><td> Fetch details of all reference assemblies. <br> Available: GrCh37, GRch38.</td><td><div class="myBtn_multi"><a href="#api_assembly_list">Click here</a></div></td></tr>
+							   <tr><td><a href="http://tark.ensembl.org/api/#!/assembly/assembly_read">GET /api/assembly/{assembly_id}/ </a> </td><td> Fetch details of a given reference assembly.</td><td><div class="myBtn_multi"><a href="#api_assembly_read">Click here</a></div></td></tr>
+								<!-- Modal for GET /api/assembly/-->
+								<div class="modal modal_multi">
+									<div class="modal-content">
+										<span class="close close_multi">&times;</span>
+										<!-----Example request - BEGIN -------------------------->
+										<div class="container">
+											<h3>Example requests for GET /api/assembly/</h2>
```

Comment:
typo: the `<h3>` tag is not closed properly, there is `</h2>` instead
this typo is replicated many times in the code down below